magma/lib/magma.rb

```diff
@@ -1,4 +1,5 @@
 require 'sequel'
+require 'active_support'
```

Comment:
@graft , importing ActiveSupport here but removing carrierwave from the gem, per your feedback on #222.